objects/authentication_factor.json

```diff
@@ -0,0 +1,82 @@
+{
+  "caption": "Authentication Factor",
+  "description": "The authentication factor object provides information about the factor used to authenticate in an mfa request.",
+  "name": "authentication factor",
+  "extends": "object",
+  "attributes": {
+    "factor_device": {
+      "description": "Device used to complete the authentication request.",
+      "requirement": "recommended"
+    },
+    "factor_type": {
+      "enum": {
+        "1": {
+          "caption": "SMS"
+        },
+        "2 question": {
+          "caption": "Security Question"
+        },
+        "3": {
+          "caption": "Call"
+        },
+        "4": {
+          "caption": "Time-based One-time Password (totp)"
+        },
+        "5": {
+          "caption": "Push Factor"
```

Comment:
If all of these are factors, should this just be "Push"?  Else, why not stick "Factor" on the end of all of them?  If we need to clarify more about "Push" maybe there could be other words like "Mobile Push" or "Push Notification" or something?

Reply:
updated to push notification